lib/tasks/exports.rake

```diff
@@ -75,4 +75,81 @@ namespace :exports do
       # scp -P 2222 -o StrictHostKeyChecking=no -o User=cf:${GUID}/0 ssh.london.cloud.service.gov.uk:/app/payable_declarations.csv .
     end
   end
+
+  desc "ECF declarations which have not moved from the submitted state (post duplicates) - [CPDLP-2032]"
+  task ecf_declarations_not_moved_from_submitted_extract: :environment do
+    headers = [
+      "Participant ID",
+      "Participant Name",
+      "TRN",
+      "Type",
+      "Schedule",
+      "Eligible for Funding",
+      "School URN",
+      "School Name",
+      "Lead Provider Name",
+      "Delivery Partner Name",
+      "Training Status",
+      "Training Status Reason",
+      "Participant Cohort",
+      "Declaration ID",
+      "Declaration Type",
+      "Declaration Date",
+      "Declaration State",
+      "Declaration Created At",
+      "Declaration Cohort",
+    ]
+
+    file_path = Rails.root.join("tmp", "submitted_declarations_#{Time.zone.today.iso8601}.csv").to_s
+
+    CSV.open(file_path, "w", headers: true) do |csv|
+      csv << headers
+      puts ">>> Initializing Report"
+
+      ParticipantDeclaration::ECF.submitted.find_in_batches do |pds|
+        pds.map do |pd|
+          pd.participant_profile
+          cpd_lead_provider = pd.cpd_lead_provider
+
+          relevant_induction_record = Induction::FindBy.call(
```

Comment:
I think might not need all this data, I think we just need the declaration id, and participant id as well as the numbers to check a few manually?